csharp/ql/test/experimental/Security Features/JsonWebTokenHandler/delegation-test.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.JsonWebTokens;
+
+namespace WilsonTest
+{
+    public class Wilson_03
```

Comment:
Why is this class named `Wilson...`?

Reply:
Internal Codename... Thanks for letting me know, I will replace the name